docs/user-guide/layer-settings.md

```diff
@@ -1,120 +1,187 @@
+!!!note
+    The Style Editor tool, that allow the user to create a new style or edit/remove an existing one, provides functionalities on top of [GeoServer](http://geoserver.org/) REST APIs. This relationship implies that this tool:
 
-* **Click** on the **Back to style list** button <img src="../img/button/style_editor_back_button.jpg" class="ms-docbutton"/> to go back without saving the changes.
+    * Is available only for those layers that are loaded in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) from [GeoServer](http://geoserver.org/)
 
-***Create new styles***
+    * Can be used only by Admin users because, by default, the GeoServer's REST APIs are available only for the GeoServer administrators. Take a look at the [User Integration with GeoServer](../../developer-guide/integrations/users/geoserver/) section of [Developer Guide](../../developer-guide) in order to understand how to configure the way MapStore and GeoServer share users, groups and roles
 
-* **Click** on the **Create new style** button <img src="../img/button/style_editor_new_style_button.jpg" class="ms-docbutton"/> if you want to create your style from scratch.   [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) shows you some style templates from which you can start your customization. Those templates can be:
+### Create a new style 
 
-    * ***CSS - Cascading Style Sheet*** (a language used for describing the presentation of a document written in a markup language like the HTML)
+It is possible to create a new style with a click on the <img src="../img/button/style_editor_new_style_button.jpg" class="ms-docbutton"/> button. At this stage the user can choose between different types of template from which the customization will start:
 
-    * ***SLD - Styled Layer Descriptor*** (an XML schema specified by the [Open Geospatial Consortium (OGC)](http://www.opengeospatial.org/) for describing the appearance of map layers)
+* *CSS - Cascading Style Sheet* (a language used for describing the presentation of a document written in a markup language like the HTML)
 
-<img src="../img/layer-settings/style_editor_add_style_template.jpg" class="ms-docimage"  style="max-width:500px;">
+* *SLD - Styled Layer Descriptor* (an XML schema specified by the [Open Geospatial Consortium OGC](http://www.opengeospatial.org/) for describing the appearance of map layers)
 
-* **Select** a style template
+<img src="../img/layer-settings/style_editor_add_style_template.jpg" class="ms-docimage" style="max-width:500px;">
 
-* **Click** on the **Add selected template to list of styles** button <img src="../img/button/style_editor_add_style_button.jpg" class="ms-docbutton"/>
+!!!note
+    The availability of the style formats depends, firstly, from the [GeoServer](http://geoserver.org/). [MapStore](https://mapstore.geo-solutions.it/mapstore/#/), by default, will add all the supported format that the server provides.
```

Comment:
To edit or create styles using the CSS format the [CSS extension](https://docs.geoserver.org/latest/en/user/styling/css/install.html) must be installed in GeoServer